src/index.js

```diff
@@ -1,14 +1,49 @@
-// eslint-disable-next-line no-unused-vars
-import _ from 'lodash';
+/* eslint-disable no-unused-vars */
```

Comment:
- [ ] Try not to disable linters, these are here to help writing good code and follow best practices.

Reply:
This issue needs further changes.
- [ ] Please ensure to add lines to your script file which will disable the linters.

Reply:
Hello bro what should I do I didn't understand the issue given here and this is not even part of the requirement now I've lost 2 chances for nothing one for forgetting about documentation and second for this issue which i'm not sure what should I about it. Also please can you can confirm that's all the issues with the project because i'm afraid that I resubmit fixing this particular issue and then I have to repeat the week for this. please help me bro

Reply:
@rloterh @Dipeshtwis

Reply:
Hi @AmineHLub,

One of the requirements of the project is to write code with best practices. And not to disable eslint manually, instead try improving your code so that you don't need to disable eslint manually and that's what we are expecting. 

Please let me know if anything is not clear here.

Thanks